infrastructure/terraform/modules/backend-api/locals.tf

```
@@ -1,7 +1,9 @@
 locals {
   csi = "${var.csi}-${var.module}"
 
-  lambdas_source_code_dir = abspath("${path.module}/../../../../lambdas")
+  monorepo_root           = abspath("${path.module}/../../../..")
```

Monorepo? naming revision please repo_root is sufficient
```suggestion
  repo_root           = abspath("${path.module}/../../../..")
```